openexr-sys/c_wrapper/cexr.cpp

```diff
@@ -203,6 +204,51 @@ void CEXR_FrameBuffer_insert(CEXR_FrameBuffer *fb,
     reinterpret_cast<FrameBuffer *>(fb)->insert(name, Slice(static_cast<Imf::PixelType>(type), base, xStride, yStride, xSampling, ySampling, fillValue, xTileCoords, yTileCoords));
 }
 
+int CEXR_FrameBuffer_get_channel(const CEXR_FrameBuffer *frame_buffer, const char name[], CEXR_Channel *out, const char **err_out) {
```

Comment:
I don't think having an `err_out` parameter is useful here, since it's very much semantically just an Option. Same goes for the Header version.